iina/Base.lproj/CropSettingsViewController.xib

```diff
@@ -76,18 +76,38 @@
                         <action selector="doneBtnAction:" target="-2" id="y1L-6G-8iz"/>
                     </connections>
                 </button>
+                <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="99r-Th-YSz">
+                    <rect key="frame" x="360" y="3" width="74" height="29"/>
+                    <constraints>
+                        <constraint firstAttribute="width" constant="62" id="HPh-UH-exv"/>
+                        <constraint firstAttribute="height" constant="18" id="IaY-oN-WAf"/>
+                    </constraints>
+                    <buttonCell key="cell" type="push" title="Cancel" bezelStyle="rounded" alignment="center" borderStyle="border" imageScaling="proportionallyDown" inset="2" id="BtO-vw-tiy">
+                        <behavior key="behavior" pushIn="YES" lightByBackground="YES" lightByGray="YES"/>
+                        <font key="font" metaFont="system"/>
+                        <string key="keyEquivalent" base64-UTF8="YES">
+Gw
+</string>
+                    </buttonCell>
+                    <connections>
+                        <action selector="cancelBtnAction:" target="-2" id="L9R-GV-vYv"/>
+                    </connections>
+                </button>
```

Comment:
The "Cancel" button should better have the same width as "Done" button, or it will looks strange in Chinese.

However you DON'T NEED to change this. I'll add/fix auto layout related stuff later.